packages/design-system-scripts/README.md

```diff
@@ -0,0 +1,39 @@
+This package contains shared scripts and tooling for CMS design systems. It includes scripts for the following:
+
+- Compiling & minifying design system files
+- Building & watching documentation site files
+- Running unit & e2e tests
+- Running eslint and stylelint
+
+## Installation
+
+This package is intended to be used with the `design-system-docs` package.
+
+```
+npm install --save @cmsgov/design-system-docs
+npm install --save @cmsgov/design-system-scripts
+```
+
+## Usage
+
+Run the design system scripts using the `cmsds` command:
+
+```
+yarn cmsds
+npx cmsds
+```
+
+Use the `--help` option to see the full list of available commands, or a command’s options
+
+```
+yarn cmsds --help
```

Comment:
I think we should list out the options here instead of making folks look to see what the options are

- `yarn cmsds build` - Builds the JavaScript and Sass for your main design-system package.
- `yarn cmsds build-docs` - Builds your main design-system package and its corresponding documentation site.
- `yarn cmsds start` - Builds and hosts the docs site locally with a webpack dev server, watching for changes in either the design-system source package or the docs package and rebuilding and refreshing appropriately.
- `yarn cmsds test` - Runs tests in one or more directory.
-  `yarn cmsds test:e2e` - Runs end to end tests in one or more directory.
- `yarn cmsds lint` -  Runs prettier, stylelint and eslint on one or more directories.